rasa/nlu/classifiers/diet_classifier.py

```diff
@@ -658,68 +766,140 @@ def _use_default_label_features(self, label_ids: np.ndarray) -> List[FeatureArra
 
     def _create_model_data(
         self,
-        training_data: List[Message],
-        label_id_dict: Optional[Dict[Text, int]] = None,
-        label_attribute: Optional[Text] = None,
+        messages: List[Message],
+        *,
```

Comment:
This is to force me naming parameters there (for an explanations see, e.g [this,](https://www.python.org/dev/peps/pep-3102/)) cause I had some bugs in intermediate version because of that. I can remove that.

Reply:
I mean we either make it a consistent practice across the code (which means it goes in engineering guidelines) or use it only when necessary as `*args`. Using it in one specific function brings more doubt in my mind as a reader.

Reply:
I just removed it to avoid confusion for now. In general, I think it is a nice tool.